multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -137,8 +137,6 @@ coreos:
         [Unit]
         Description=decrypt kubelet tls assets using amazon kms
         Before=kubelet.service
```

Comment:
@robszumski also keep in mind that ordering constraints in systemd are entirely decoupled from dependency relationships. 

Even in the absence of any explicit `Required/RequiredBy` (or equivalent) relationship between the two services, If both services were being brought up as dependencies of something else, the `Before/After` relationships between them would still be applied even though the two services do not depend on each other in any way.